includes/payments/class-wc-payments-controller.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * REST API Payments Controller
+ *
+ * Handles requests to the /payments endpoint.
+ *
+ * @author   Automattic
+ * @category API
+ * @package  WooCommerce/API
+ */
+
+use Automattic\WooCommerce\Admin\Notes\Note;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * REST API Settings controller class.
+ *
+ * @package WooCommerce/API
+ */
+class WC_Payments_Controller extends WC_REST_Controller {
+
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wc-calypso-bridge/v1';
+
+	/**
+	 * Route base.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'payments';
+
+	/**
+	 * Register routes.
+	 */
+	public function register_routes() {
+		register_rest_route( $this->namespace, '/' . $this->rest_base . '/activate-promo', array(
+			array(
+				'methods'             => 'POST',
```

Comment:
I changed the API name from `post-install` to `activate-promo` and methods to `POST` as well.